site/docs/v1/tech/core-concepts/scim.adoc

```diff
@@ -170,6 +170,38 @@ In order to use FusionAuth as your SCIM provisioning server for SCIM clients, yo
 
 See the link:/docs/v1/tech/apis/scim/[SCIM API Overview] for details about the supported endpoints.
 
+== Client SCIM Authentication
+
+FusionAuth requires a completed Client Credentials grant and the resulting token for SCIM server authentication, as mentioned above. The token generated from the grant is used in the `Authorization` header as a `Bearer` token.
+
+However, some SCIM clients can't dynamically complete a client credentials grant. Instead, they need a static authorization header value.
+
+To correctly integrate such clients, do the following:
+
+* Change the JWT duration of the SCIM Client Entity Type to a larger value, such as a year, which is `31536000`.
```

Comment:
Should this have a qualifier to say something like - "as long as you are comfortable with knowing that this token cannot be revoked"?

Reply:
Sure, I added a note.